src/System.Windows.Extensions/src/Interop/Windows/Interop.CertAddCertificateLinkToStore.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+using System.Security.Cryptography.X509Certificates;
+
+internal static partial class Interop
+{
+    internal static partial class Crypt32
+    {
+        internal const uint CERT_STORE_ADD_ALWAYS = 4;
+
+        [DllImport(Interop.Libraries.Crypt32, CharSet = CharSet.Unicode, SetLastError = true)]
+        internal extern static bool CertAddCertificateLinkToStore(SafeCertStoreHandle hCertStore, SafeCertContextHandle pCertContext, uint dwAddDisposition, [In, Out] SafeCertContextHandle ppStoreContext);
+    }
+}
```

Comment:
Is there a reason we're not putting all of these interop files under src/Common/src/Interop/Windows?

Reply:
they are not being used anywhere else. Different overloads of these functions are in ```System.Security.Cryptography.X509Certificates```

Reply:
> they are not being used anywhere else.

They don't need to be.  In general all of our interop definitions should go into src/Common/src/Interop/.